### ISSUE TITLE
Toga-winforms says not installed but i have it installed

### ISSUE THREAD

**Issue by frehburg** [beeware/toga#1595]
**Describe the bug**
My program is complaining that toga-winforms is not installed even tough it is clearly installed.

C:\Users\filre\anaconda3\python.exe C:/Users/filre/OneDrive/Documents/DataSpell/HongKongSites/bw/hksites/src/hksites/__main__.py
Traceback (most recent call last):
  File "C:/Users/filre/OneDrive/Documents/DataSpell/HongKongSites/bw/hksites/src/hksites/__main__.py", line 4, in <module>
    main().main_loop()
  File "C:\Users\filre\OneDrive\Documents\DataSpell\HongKongSites\bw\hksites\src\hksites\app.py", line 66, in main
    return Hksites('Hong Kong Sites','com.filreh')
  File "C:\Users\filre\anaconda3\lib\site-packages\toga\app.py", line 278, in __init__
    self.factory = get_platform_factory(factory)
  File "C:\Users\filre\anaconda3\lib\site-packages\toga\platform.py", line 53, in get_platform_factory
    from toga_winforms import factory
ModuleNotFoundError: No module named 'toga_winforms'
**To Reproduce**
This is my source written by following: https://www.youtube.com/watch?v=xezYYBL7nk0
```"""
This project will hold a list of things i wnat to do in Hong Kong.
"""
import toga


class Hksites(toga.App):

    def startup(self):
        """
        Construct and show the Toga application.

        Usually, you would add your application to a main content box.
        We then create a main window (with a name matching the app), and
        show the main window.
        """
        box = toga.Box()
        self.f_input = toga.TextInput()
        f_label = toga.Label('Fahrenheit', alignment=toga.LEFT_ALIGNED)

        join_label = toga.Label('is equivalent to', alignment=toga.RIGHT_ALIGNED)

        self.c_input = toga.TextInput(readonly=True)
        c_label = toga.Label('Celsius', alignment=toga.LEFT_ALIGNED)

        button = toga.Button('Calculate',on_press=self.calculate)

        f_box = toga.Box()
        c_box = toga.Box()

        f_box.add(self.f_input)
        f_box.add(f_label)

        c_box.add(join_label)
        c_box.add(self.c_input)
        c_box.add(c_label)

        box.add(f_box)
        box.add(c_box)
        box.add(button)

        box.style.set(flex_direction='column',padding=10)
        f_box.style.set(flex_direction='row',margin=5)

        self.f_input.style.set(flex=1,margin_left=160)
        f_label.style.set(width=100,margin_left=10)

        self.c_input.style.set(flex=1)
        c_label.style.set(width=100,margin_left=10)
        join_label.style.set(width=150,margin_right=10)

        button.style.set(margin_top=15)

        self.main_window = toga.MainWindow(title=self.formal_name)
        self.main_window.content = box
        self.main_window.show()


    def calculate(self,widget):
        try:
            self.c_input.value = (float(self.f_input.value)-32.0)*5.0/9.0
        except Exception:
            self.c_input.value = '???'

def main():
    return Hksites('Hong Kong Sites','com.filreh')
```

**Expected behavior**
Just run lol.

**Screenshots**
See that toga-winforms is installed:
![See that toga-winforms is installed](https://user-images.githubusercontent.com/43171336/192745961-766fe874-7818-4065-b27b-55c3c7c0406c.png)

But it's still complaining:
![But it's still complaining](https://user-images.githubusercontent.com/43171336/192746076-553a81d8-4277-4fef-ac95-c50f130d17ef.png)

**Environment:**
 - Operating System: win11
 - Python version: 3.8.8
 - Software versions:
   - Briefcase: 0.3.9
   - Toga: 0.3

**Comment by frehburg** [comment]
I am usually using anaconda to install packages but used pip inside of my conda terminal to install toga, toga-winform and briefcase using the 
```pip install --pre toga```
```pip install toga-winforms```
...
commands

**Comment by sbbosco** [comment]
The wheel file **toga_winforms-0.3.0.dev37-py3-none-any.whl** is missing the **toga_winforms** folder.

**Comment by mhsmith** [comment]
Sorry, it looks like something went wrong with today's Toga release. I'm looking into it just now.

**Comment by mhsmith** [comment]
For now, you can work around the problem by editing your pyproject.toml file to find the line that looks like this (your version number may be different):
```
    'toga-winforms>=0.3.0.dev32',
```
and change it to this (note `>=` changed to `==`):
```
    'toga-winforms==0.3.0.dev36',
```

Then run `briefcase update -d` before running the app again.

**Comment by rmartin16** [comment]
FWIW, I don't think there's a problem in the code, When I manually build the `toga-winforms` wheel from `main` and install it (along with wheels built for `toga-core` and `toga-dummy`), everything works fine.

**Comment by freakboy3742** [comment]
Well this is just weird. The required files are all in the *source* tarball, so a source install will work - it's just the wheel that is borked. Looks like 4ffff033 has... not worked.

**Comment by freakboy3742** [comment]
I've just reverted the offending change, and published 0.3.0dev38. Thanks for the report, and apologies for the inconvenience.
